app/components/sbom/app-list/action/index.ts

```diff
@@ -0,0 +1,91 @@
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import IntlService from 'ember-intl/services/intl';
+
+import { AkIconSignature } from 'irene/components/ak-icon';
+import SbomAppModel from 'irene/models/sbom-app';
+import { SbomScanStatus } from 'irene/models/sbom-scan';
+
+export interface SbomAppListActionSignature {
+  Args: {
+    sbomApp: SbomAppModel;
+    onViewReportClick: (sbomApp: SbomAppModel) => void;
+  };
+}
+
+type MenuItem = {
+  label: string;
+  icon: string;
+  iconVariant?: AkIconSignature['Args']['variant'];
```

Comment:
Can you create a menu item variant that extends/inherits from the AkIcon's icon variant?